server.lua

```diff
@@ -178,17 +197,13 @@ local function openInventory(source, invType, data, ignoreSecurityChecks)
 				right = Inventory(('evidence-%s'):format(data))
 			end
 		elseif invType == 'dumpster' then
-			---@cast data string
-			right = Inventory(data)
+			local dumpsterId = getDumpsterFromCoords(data)
+			right = dumpsterId and Inventory(('dumpster-%s'):format(dumpsterId))
 
 			if not right then
-				local netid = tonumber(data:sub(9))
-
-				-- dumpsters do not work with entity lockdown. need to rewrite, but having to do
-				-- distance checks to some ~7000 dumpsters and freeze the entities isn't ideal
```

Comment:
Just going to point at

> freeze the entities isn't ideal

Reply:
The distance check to ~7000 dumpsters has been avoided by only doing it to those that have been interacted with, unfortunately there isn't really a way to get around this without freezing them but I will implement the convar as suggested.